src/xunit.performance.api/PerformanceMetric/GCAllocatedBytesForCurrentThreadMetric.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.Diagnostics.Tracing;
+using Microsoft.Diagnostics.Tracing.Parsers;
+using Microsoft.Xunit.Performance.Sdk;
+using System.Collections.Generic;
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    internal class GCAllocatedBytesForCurrentThreadMetric : PerformanceMetric
+    {
+        public GCAllocatedBytesForCurrentThreadMetric()
+            : base("GCAllocatedBytesForCurrentThread", "Allocations In Current Thread", PerformanceMetricUnits.Bytes)
```

Comment:
I think we may want to name the metric differently, since I'm not sure it will be clear to the viewer of data what "Current Thread" is.  Maybe "Allocation Size on Benchmark Execution Thread" or something like that....